src/polybench/POLYBENCH_3MM-Cuda.cpp

```diff
@@ -143,12 +143,13 @@ void POLYBENCH_3MM::runCudaVariant(VariantID vid)
     using EXEC_POL =
       RAJA::KernelPolicy<
         RAJA::statement::CudaKernelAsync<
-          RAJA::statement::For<0, RAJA::cuda_block_exec,
-            RAJA::statement::For<1, RAJA::cuda_thread_exec,
+          RAJA::statement::For<0, RAJA::cuda_block_x_loop,
+            RAJA::statement::For<1, RAJA::cuda_thread_y_loop,
```

Comment:
Should this use cuda_block_y_loop, and cuda_thread_x_loop to more consistently use thread_x for adjacent threads.

Reply:
Yes, good catch! Thanks.

Reply:
Actually, I take that back. The RAJA variant matches the base CUDA variant.

Reply:
But, I will make the change you suggested here and in other kernels.  Thanks.